lib/b2b/discovery.ts

```diff
@@ -57,22 +57,27 @@ class Organizations {
     this.fetchConfig = fetchConfig;
   }
 
-  list(data: OrganizationsRequest): Promise<OrganizationsResponse> {
-    return request<OrganizationsResponse>(this.fetchConfig, {
+  list(
+    data: B2BDiscoveryOrganizationsRequest
+  ): Promise<B2BDiscoveryOrganizationsResponse> {
+    return request<B2BDiscoveryOrganizationsResponse>(this.fetchConfig, {
       method: "POST",
       url: "discovery/organizations",
       data,
     });
   }
 
   create(
-    data: DiscoveryOrganizationCreateRequest
-  ): Promise<DiscoveryOrganizationCreateResponse> {
-    return request<DiscoveryOrganizationCreateResponse>(this.fetchConfig, {
-      method: "POST",
-      url: "discovery/organizations/create",
-      data,
-    });
+    data: B2BDiscoveryDiscoveryOrganizationCreateRequest
+  ): Promise<B2BDiscoveryDiscoveryOrganizationCreateResponse> {
+    return request<B2BDiscoveryDiscoveryOrganizationCreateResponse>(
```

Comment:
Is this double `Discovery` correct? In this case would $Method be DiscoveryOrganizationCreate?

Reply:
Good catch - with ~160 renames I was bound to mess a few up :)